src/api/flaskr/service/tts/streaming_tts.py

```
@@ -0,0 +1,453 @@
+                    audio_bid=self._audio_bid,
+                    duration_ms=final_duration_ms,
+                ),
+            )
+
+            logger.info(
+                f"TTS complete: audio_bid={self._audio_bid}, "
+                f"segments={len(audio_data_list)}, "
+                f"duration={final_duration_ms}ms"
+            )
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)